app/models/cart.rb

```diff
@@ -1,3 +1,4 @@
+require_relative 'cart_coffee'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.